packages/financial-templates-lib/src/proxy-transaction-handler/DSProxyManager.js

```diff
@@ -139,16 +146,20 @@ class DSProxyManager {
     });
 
     await this.gasEstimator.update();
-    const executeTransaction = await this.dsProxy.methods["execute(bytes,bytes)"](callCode, callData).send({
-      from: this.account,
-      gasPrice: this.gasEstimator.getCurrentFastPrice()
+    const executeTransaction = await runTransaction({
+      transaction: this.dsProxy.methods["execute(bytes,bytes)"](callCode, callData),
+      config: {
+        gasPrice: this.gasEstimator.getCurrentFastPrice(),
+        from: this.account,
+        nonce: await this.web3.eth.getTransactionCount(this.account)
```

Comment:
None of these `runTransaction` methods are getting called in parallel with each other right? Otherwise the `nonce` is going to be problematic. It doesn't look like an issue but just want to double check